internal/pkg/state/view.go

```diff
@@ -170,6 +183,18 @@ func (v *View) MinerGetPrecommittedSector(ctx context.Context, maddr addr.Addres
 	return minerState.GetPrecommittedSector(StoreFromCbor(ctx, v.ipldStore), abi.SectorNumber(sectorNum))
 }
 
+// MarketEscrowBalance looks up a token amount in the escrow table for the given address
+func (v *View) MarketEscrowBalance(ctx context.Context, addr addr.Address) (found bool, amount abi.TokenAmount, err error) {
```

Comment:
This is needed to test if we need to create and escrow balance in the market actor for a deal participant.